### PR TITLE
Add unit tests for pipelinerun

### PR DESCRIPTION
**Pull request by UtkarshChaurasia**
- Implement delete function for mkspipelinerun controller
-  Add unit tests for pipelinerun create, delete and convert to tekton

### How to test

- Run this command in the project folder
```bash
    go test ./...
```

### CODE REVIEW

**Review by ramessesii2 — APPROVED** [review]
lgtm

File: pkg/controllers/mkspipelinerun/mks-pipelinerun.go [hunk 1]
```diff
@@ -51,16 +51,24 @@ func Create(cl *tconfig.Client, mpr *v1alpha1.MksPipelineRun, opt metav1.CreateO
...
+
 // Add method of controller for informer function: AddFunc
 func (c *controller) handleAdd(obj interface{}) {
 	fmt.Println("add was called")
 	fmt.Println(obj)
 	tp := &tconfig.TektonParam{}
 	cs, err := tp.Client()
 	if err != nil {
-		fmt.Errorf("Cannot get client", err)
+		fmt.Errorf("Cannot get client %s", err)
```
Comment (avinal) [inline pkg/controllers/mkspipelinerun/mks-pipelinerun.go]:
```suggestion
		fmt.Errorf("Cannot get client %v", err)
```

File: pkg/controllers/mkspipelinerun/mkspipelinerun_test.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+			Namespace: "default",
+		},
+	}
+
+	expected.Kind = "PipelineRun"
+	expected.APIVersion = "tekton.dev/v1beta1"
+	fmt.Println(expected.Kind, " ", converted.Kind)
+	fmt.Println(expected.Spec.PipelineRef.Name, " ", converted.Spec.PipelineRef.Name)
+	fmt.Println(expected.ObjectMeta.Name, " ", converted.ObjectMeta.Name)
+	fmt.Println(expected.ObjectMeta.Namespace, " ", converted.ObjectMeta.Namespace)
```
Comment (avinal) [inline pkg/controllers/mkspipelinerun/mkspipelinerun_test.go]:
```suggestion
```
Not needed

**Review by avinal — APPROVED** [review]
Looks nice Now

**Pull request merged**